jobs/nfcens/prep/JEVS_NFCENS_WAVE_GRID2OBS_PREP

```diff
@@ -65,26 +58,23 @@ export VERIF_CASE=${VERIF_CASE:-grid2obs}
 # Define COMIN/COMOUT variables     
 ####################################
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
-export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${model_ver})} 
+export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})} 
 export COMINgdas=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}
 
-export COMOUT=${COMOUT:-$(compath.py -o $NET/$evs_ver)}
-export ARCmodel=${COMOUT}/$STEP/$COMPONENT/${RUN}.${INITDATE}/${MODELNAME}/${VERIF_CASE}
+export COMOUT=${COMIN}/${STEP}/${COMPONENT}/${RUN}
```

Comment:
@lgannoaa , this line is troublesome. The COMOUT variable needs to be able to be overwritten from the driver script. Can this be changed to something like this?
`export COMOUT=${COMOUT:-${COMIN}/${STEP}/${COMPONENT}/${RUN}}`

Reply:
Modified as requested.